.github/workflows/ci.yml

```diff
@@ -1,51 +1,16 @@
-name: Continuous Integration
-
+name: CI
 on:
   push:
     branches: [ main ]
   pull_request:
     branches: [ main ]
+  workflow_dispatch:
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    env:
-      DEBUG: True
-      DATABASE_URL: postgis://postgres:postgres@localhost/geo-search
-      DJANGO_SECRET_KEY: test-secret
-      PYTHON_VERSION: "3.11"
-    steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python 3.11
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ env.PYTHON_VERSION }}
-    - name: Install required Ubuntu packages
-      run: |
-        sudo apt-get install gdal-bin
-    - name: Install PyPI dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-        pip install -r requirements-dev.txt
-    - name: Run Python side code neatness tests
-      run: |
-        flake8
-        black --check .
-        isort . -c
-    - name: Run pytest code functionality tests
-      run: |
-        pytest --cov=.
-
-    services:
-      postgres:
-        image: postgis/postgis:15-3.3
-        env:
-          POSTGRES_HOST_AUTH_METHOD: trust
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          - 5432:5432
+  common:
+    uses: City-of-Helsinki/.github/.github/workflows/ci-django-api.yml@main
+    secrets: inherit
+    with:
+      python-version: "3.12"
+      postgres-major-version: 13
```

Comment:
Is this correct? The compose file uses version 16.

Reply:
Updated to use Postgres version 16.